transcribed/page0280.txt

```diff
@@ -1,7 +1,7 @@
 001                                                                             PAGE 28
 002             ;SUBTTL CALL DISPATCH
 003
-004     001 017 ACALL:  LDB T,[270400,40
+004     001 017 ACALL:  LDB T,[270400,,40
```

Comment:
Good catch!  Even if it's hard to see, it must be two commas here.  We know this from context: LDB is a byte instruction and 270400,, specifies a bit field from memory location 40.